src/mesh.cpp

```diff
@@ -446,6 +445,13 @@ scene::IMesh* convertNodeboxesToMesh(const std::vector<aabb3f> &boxes,
 				box.MinEdge.Z=box.MaxEdge.Z;
 				box.MaxEdge.Z=temp;
 			}
+		
+		box.MinEdge.X -= expand;
+		box.MinEdge.Y -= expand;
+		box.MinEdge.Z -= expand;
+		box.MaxEdge.X += expand;
+		box.MaxEdge.Y += expand;
+		box.MaxEdge.Z += expand;
```

Comment:
yes there is, but me and kahrl have agreed that box.MinEdge += expand would be misleading
one is v3f second is float